### PR TITLE
[PSD-2356] Update folders

### PR DESCRIPTION
**Pull request by alexburley**

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
...hould be able to easily connect together their entire cloud software stack and automate complex processes in order to scale their growth. \n\nCitizen automators in marketing, sales, support and business operations love Tray’s unique solution.\n\n* Connect any web application with Tray’s unique Universal Connector.\n* Get started in minutes with pre-built automations for many popular workflows. \n* Easy drag-and-drop interface that doesn’t require dev resources.\n* Go beyond simplistic point-to-point integrations with our flexible conditional logic operators that handle complex multi-step, multi-app workflows.\n* No limits. No latency. Powerful processing of billions of tasks in real-time with the power of elastic serverless architecture. \n* Prompt and helpful support to ensure your success.\n\nTray.io is a cloud data integration and automation platform built for business people at enterprises and fast-growing companies. Try it out by [signing up for a free trial](https://tray.io/lp/zendesk/trial?utm_medium=partner&utm_source=directory&utm_campaign=zendesk) or [see a personalized demo](https://tray.io/lp/zendesk/demo?utm_medium=partner&utm_source=directory&utm_campaign=zendesk). ",
```
Comment (ajstone71) [inline app/translations/en.json]: Not sure why the long_description and installation_instructions are outside of the blob? Did the compiler fail without placing them there? Please see: https://developer.zendesk.com/apps/docs/developer-guide/multiproduct_apps

**Review by ajstone71 — APPROVED** [review]
All of this looks good, need to check on the comment for the en.json file to make sure it is formatted properly

**Review by ajstone71 — CHANGES_REQUESTED** [review]
Please review the en.json file for content before merging. Please see docs here: https://developer.zendesk.com/apps/docs/developer-guide/multiproduct_apps

They give an example at the end of the page

File: app/translations/en.json (hunk 1 above)
Comment (alexburley) [reply app/translations/en.json]: Good spot

**Review by jdon — APPROVED** [review]

**Pull request merged**